main.go

```diff
@@ -163,6 +164,19 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 					return
 				}
 			}
+		case botPrefix + "weather":
+			var apiKey = "your apikey here"
```

Comment:
Small change: Name the apiKey variable aptly (include openweather in the name) so we know what apiKey this is. Define the api key at the start of main.go, along with the discord bot token so we remember to put this in the env file when working on issue #7

Reply:
Let me know if the changes are good